Bangazon/Bangazon.csproj

```diff
@@ -1,16 +1,18 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
     <UserSecretsId>aspnet-Bangazon-B718BE39-0087-4330-A24E-889383886922</UserSecretsId>
     <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
   </PropertyGroup>
 
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
-    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.2.3" />

```

Comment:
@askingalot Why delete the CodeGeneration.Design package? I believe this is still needed to be able to scaffold

Reply:
I can't believe I missed this comment. I'm sorry about that.

Looks like visual studio will add that reference back when you use scaffolding the first time. I updated the project file and added the two dependencies that VS informed me should be there.

Reply:
@AdamSheaffer Will you please re-review?